src/main/java/com/gregtechceu/gtceu/api/machine/trait/RecipeLogic.java

```diff
@@ -567,10 +567,11 @@ public void updateSound() {
             }
             if (sound != null) {
                 workingSound = sound.playAutoReleasedSound(
-                        () -> machine.shouldWorkingPlaySound() && isWorking() && !getMachine().isInValid() &&
-                                getMachine().getLevel().isLoaded(getMachine().getPos()) &&
-                                MetaMachine.getMachine(getMachine().getLevel(), getMachine().getPos()) == getMachine(),
-                        getMachine().getPos(), true, 0, 1, 1);
+                        () -> machine.shouldWorkingPlaySound() && isWorking() && !getMachine().isRemoved() &&
```

Comment:
This is quite something...

Your PR just renamed stuff to `getBlockPos()` here, but this really needs to be made saner at some point.